File: RecycleTest/app/src/main/java/com/example/pch/recycletest/MainActivity.java
package com.example.pch.recycletest;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import androidx.recyclerview.widget.DividerItemDecoration;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;
import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    private ArrayList<String> mArrayList;
    private CustomAdapter mAdapter;
    private RecyclerView mRecyclerView;
    private LinearLayoutManager mLinearLayoutManager;
    private String[] items = {};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mRecyclerView = findViewById(R.id.view);
        mLinearLayoutManager = new LinearLayoutManager(this);
        mRecyclerView.setLayoutManager(mLinearLayoutManager);

        mArrayList = new ArrayList<>();
        for (int i = 0; i < items.length; i++) {
            mArrayList.add(items[i]);
        }
        mAdapter = new CustomAdapter(mArrayList);
        mRecyclerView.setAdapter(mAdapter);
    }
}

File: CustomView3/app/src/main/java/com/example/pch/customview/MainActivity.kt
package com.example.pch.customview

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
//        Log.d("visible", "${View.VISIBLE}")
//        Log.d("visible", "${View.GONE}")
//        val sview = findViewById<PaySwitchMenu>(R.id.customSwitch)
//
//        val handler = Handler()
//        handler.postDelayed( { sview.toggle() },1000)
//
        val nview = findViewById<PaySwitchMenu>(R.id.customSwitch2)
    }
}
